pkgs/development/python-modules/kaleido/default.nix

```
@@ -0,0 +1,77 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchurl,
+  autoPatchelfHook,
+  expat,
+  libGL,
+  nspr,
+  nss,
+  sqlite,
+  system ? builtins.currentSystem,
+}:
+
+buildPythonPackage rec {
+  pname = "kaleido";
+  version = "0.2.1.post1";
```

```suggestion
  version = "0.2.1";
```
This version just adds support for arm32.
https://github.com/plotly/Kaleido/releases